src/taskgraph/main.py

```diff
@@ -439,6 +439,7 @@ def show_taskgraph(options):
         diffcmd = [
             "diff",
             "-U20",
+            "--color=always",
```

Comment:
Hm, I think this will break piping the output into other commands which I do on occasion. It may also send control characters if redirecting into a file (like the `tgdiff` task does in Gecko).

To work around this in Gecko I created a setting that people can add to their `~/.machrc` file:
https://searchfox.org/mozilla-central/source/taskcluster/mach_commands.py#30

Though we don't have the benefit of that here.. Maybe we could look for an environment variable instead or something.

Reply:
You're right. I removed this line. I'll just pipe to `colordiff` on my local machine.